server/issuers.go

```diff
@@ -224,22 +225,27 @@ func (c *Server) issuerGetAllHandler(w http.ResponseWriter, r *http.Request) *ha
 			Code:    500,
 		}
 	}
-	respIssuers := []issuerResponse{}
-	for _, issuer := range *issuers {
-		expiresAt := ""
-		if !issuer.ExpiresAt.IsZero() {
-			expiresAt = issuer.ExpiresAt.Format(time.RFC3339)
-		}
 
-		var publicKey *crypto.PublicKey
-		for _, k := range issuer.Keys {
-			publicKey = k.SigningKey.PublicKey()
-		}
+	marshalledIssuers, err := utils.MarshalIssuersAndSigningKeys(issuers)
+	if err != nil {
+		c.Logger.Error("error retrieving issuer values")
+		panic(err)
+	}
 
-		respIssuers = append(respIssuers, issuerResponse{issuer.ID.String(), issuer.IssuerType, publicKey, expiresAt, issuer.IssuerCohort})
+	var respIssuers []issuerResponse
+	for _, value := range marshalledIssuers {
+		iss := value.Issuer
+		issuerKeySigningKey := value.SigningKey
+		respIssuers = append(respIssuers, issuerResponse{
+			iss.ID.String(),
+			iss.IssuerType,
+			issuerKeySigningKey.PublicKey(),
+			iss.ExpiresAt.Format(time.RFC3339),
+			iss.IssuerCohort,
+		})
```

Comment:
So with `utils.MarshalIssuersAndSigningKeys` we get a map of the issuer and it's various issuerKeys.

It seems to me that we can just return the issuer as well as it's actual signing key, not just the last one as it was previously. 

I do wonder though, if we should exclude anything that has "expired" (seems like yes).